junction/templates/profiles/dashboard.html

```diff
@@ -34,7 +34,7 @@ <h1>Your Proposals</h1>
             <br>
             {% if conf_proposals %}
                 {% for conference,proposals in conf_proposals.items %}
-                <h3><a href="{% url 'proposals-list' conference|slugify %}">{{ conference }}</a></h3>
+                <h3><a href="{% url 'proposals-list' proposals.0.conference.slug %}">{{ conference }}</a></h3>
```

Comment:
Shall we pass `conference` as the conference object itself instead of it being a string? In that case, we can directly do `conference.slug`

Reply:
Well the logic is using dict to group the proposals and uses conference name as key. So that is something needed. Conference object is used in only this one place so I thought of not passing it. If we do want it, we can have dict of tuples with conference and proposal list. But that will require more changes, This is a simple one.

Reply:
@palnabarun Any update here ?

Reply:
I think the current fix is good to go at the moment.